### PR TITLE
Extended profile usage

### PR DESCRIPTION
**Pull request by meyhane**
- `download` and `status`commands accept 'profile' argument now

### CODE REVIEW

File: tests/fair_weather_test/.mysesamconfig.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (meyhane) [inline tests/fair_weather_test/.mysesamconfig.json:1]: adding a non-default formatstyle to verify that file was effective

File: sesam.py [hunk 2]
```diff
@@ -505,17 +509,64 @@ def __init__(self, args, logger):
...
+                config = {}
+                for section in parser.sections():
+                    for key, value in parser.items(section):
+                        config[key.lower()] = value
+
+        return config
+
+
+    def read_config_file(self, filename, is_required=False):
+        try:
```
Comment (meyhane) [inline sesam.py:537]: a code from _get_node_and_jwt_token_ is extracted as method here for reusability

File: sesam.py [hunk 3]
```diff
@@ -798,6 +844,17 @@ def download(self):
         self.logger.info("Replaced local config successfully")
 
     def status(self):
+        def log_and_get_diff_flag(file_content1, file_content2, file_name1, file_name2):
```
Comment (meyhane) [inline sesam.py:847]: an earlier code block is extracted as method again

**Review by tombech — APPROVED** [review]
Looks good to me, remember to add a "2.2.0" tag as well to get a new release built

**Comment by branislavjenco** [comment]
Looks good to me too. One nitpick - I'd name the `.sesamconfig.json` file something else, maybe `.formatconfig.json` to imply it's configuring the formatting options. `.sesamconfig.json` might imply that it configures general sesam options - which is what the `.syncconfig` file does already.

**Review by branislavjenco — APPROVED** [review]

**Comment by meyhane** [comment]
Ahh. @branislavjenco  I saw your comment late.
`.syncconfig` is purely syncing, that is for node connection, not for general sesam options, isn't it?
Indeed I named it so because it can be used for any other _general_ sesam options that we want to add support for, where formatstyle is one of them(and the only one per now)

**Pull request merged**